### ISSUE TITLE
[Issue]: Audio-only transcoding with ffmpeg loses HDR information

### ISSUE THREAD

**Issue by stoically** [jellyfin/jellyfin#7295]
### Please describe your bug

Given a media file with a HVEC video track in a mkv container and an AC3 default audio track, my device is capable of direct play and will switch into HDR mode.

However, if I switch the audio track to something that my device can't play, which in this case is a DTS audio track, transcoding of just the audio track needs to happen, which in this case it does with ffmpeg. Now my device will no longer switch into HDR mode, since the information in the mkv container about it being HDR seems to get lost in the process of transcoding with ffmpeg.

I've read a bit on the web and it seems that there's some combination of parameters that one can pass to ffmpeg to preserve the HDR info bit, but I'm no expert and not sure which one actually works.

If I can help debugging further, please let me know.

Originally reported in https://github.com/jellyfin/jellyfin-webos/issues/76

### Jellyfin Version

10.7.7

### Environment

```markdown
- OS: Ubuntu Server
- Virtualization: Docker
- Clients: LG 49UN71006LB
- Browser: Firefox
- FFmpeg Version: Docker
- Playback Method: Direct Play / Transcode
- Hardware Acceleration: None
- Plugins: None
- Reverse Proxy: None
- Base URL: None
- Networking: Host
- Storage: local
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by GodTamIt** [comment]
From my testing, #6454 fixes this issue (as a side effect) if only the audio needs to be transcoded, since the video will no longer be transcoded.

If the video stream is actually incompatible, then you will still get a transcode to SDR (most likely 8-bit H.264). Whether that's an issue and we should consider conditional 10-bit H.264 support, that's up to the team.

**Comment by crobibero** [comment]
Fixed by https://github.com/jellyfin/jellyfin/issues/7295

**Comment by stoically** [comment]
> Fixed by #7295

Just tested and can confirm that it was fixed by https://github.com/jellyfin/jellyfin/pull/7325 - thanks so much!
